src/azure-cli/HISTORY.rst

```diff
@@ -10,6 +10,10 @@ Release History
 
 * `az acr check-health`: Fix "DOCKER_PULL_ERROR" on Windows (#13158)
 
+**AppService**
+
+* `az webapp up`: Fixing case for creating RG is failing when location is not provided
```

Comment:
Do not modify HISTORY.rst manually. Put it in PR title or PR description. HISTORY.rst will be generated when a new version is going to release.
https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules#submitting-pull-requests

Reply:
Got it.